### ISSUE TITLE
Import past projects from the old Observatory

### ISSUE THREAD

**Issue by JDougherty** [rcos/Observatory3#371]
RCOS has many more past projects than are shown on the [past projects page](https://rcos.io/projects). Please respect the work of alums and import data from the old Observatory, if possible.

**Comment by agundy** [comment]
@JDougherty I agree! We plan on bring the old Observatory back on line if possible. We had to take down the old Observatory because of a request from RPI about not having HTTPS on the server. We will also import the old projects into the new page.
